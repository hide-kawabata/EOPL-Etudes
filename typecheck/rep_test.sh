echo test/add.eopl
./rep_loop < test/add.eopl
echo test/add2.eopl
./rep_loop < test/add2.eopl
echo test/even.eopl
./rep_loop < test/even.eopl
echo test/if.eopl
./rep_loop < test/if.eopl
echo test/let.eopl
./rep_loop < test/let.eopl
echo test/let2.eopl
./rep_loop < test/let2.eopl
echo test/proc.eopl
./rep_loop < test/proc.eopl
echo test/proc1.eopl
./rep_loop < test/proc1.eopl
echo test/proc2.eopl
./rep_loop < test/proc2.eopl
echo test/proc3.eopl
./rep_loop < test/proc3.eopl
echo test/proc4.eopl
./rep_loop < test/proc4.eopl
echo test/proc5.eopl
./rep_loop < test/proc5.eopl
echo test/val.eopl
./rep_loop < test/val.eopl
