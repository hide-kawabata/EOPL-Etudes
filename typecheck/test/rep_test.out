test/add.eopl
eopl> 9: int
eopl> 
test/add2.eopl
eopl> 2: int
eopl> 
test/even.eopl
eopl> false: bool
eopl> 
test/if.eopl
eopl> false: bool
eopl> 
test/let.eopl
eopl> 25: int
eopl> 
test/let2.eopl
eopl> 9: int
eopl> 
test/proc.eopl
eopl> 7: int
eopl> 
test/proc1.eopl
eopl> 17: int
eopl> 
test/proc2.eopl
eopl> 16: int
eopl> 
test/proc3.eopl
eopl> 8: int
eopl> 
test/proc4.eopl
eopl> 7: int
eopl> 
test/proc5.eopl
eopl> 18: int
eopl> 
test/val.eopl
eopl> 52: int
eopl> 
